app/javascript/react/container/AirlineShowContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { browserHistory, Link } from 'react-router';
+import AirlineShowTile  from '../components/AirlineShowTile'
+
+class AirlineShowContainer extends Component {
+ constructor(props) {
+   super(props);
+   this.state= {
+     airline: "",
+     destinations: "",
+     rating: ""
+
+   }
+ }
+ componentDidMount(){
+
+ let id = this.props.params.id
+ console.log(id)
+
+ fetch(`/api/v1/airlines/${id}`)
+ .then(response => {
+   let responseBody = response.json()
+   return responseBody
+ }).then(responseBody => {
+
+   this.setState({
+     airline: responseBody.name,
+     destinations: responseBody.destinations,
+     rating: responseBody.overall_rating
+   })
+ })
+}
+
+ render(){
+   return (
+     <div>
+     <AirlineShowTile
+     key={this.props.params.id}
```

Comment:
I'd like to see different tests for the show container and the show tile -- for an example of testing containers vs tiles, check out [Testing React Components Implementation](https://learn.launchacademy.com/teams/boston-23/curricula/on-campus-boston-23/lesson_groups/week_4:_advanced_react/lessons/testing-react-components-implementation)

Reply:
noted -- thank you. all done.